dockers/actions-plan-preview/main.go

```diff
@@ -109,8 +120,22 @@ func main() {
 		}
 	}
 
+	// find comment before we send
+	comment, err := findLatestPlanPreviewComment(ctx, ghClientV4, event.Owner, event.Repo, event.PRNumber)
```

Comment:
How about finding all previous comments which are not minimized to hide instead of finding just the latest one?
Not a big deal but if we can we should do it.

Reply:
I wasn't sure about this, but I thought it might be annoying for people who have comments they don't want to hide, so I went with the one right before.

Reply:
Ah, I see your point. Nice point.
Users may re-display a comment that was hidden by our action before right.
I totally agree with you. Let's keep it as is.